### PR TITLE
Define MyAccount view for a user without local credentials

### PR DESCRIPTION
**Pull request by VivekVinushanth**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/myaccount/src/extensions/configs/scim.ts [hunk 1]
```diff
@@ -30,7 +30,8 @@ export const SCIMConfigs: SCIMConfigInterface = {
         core1Schema: "urn:scim:schemas:core:1.0",
         coreSchema: "urn:ietf:params:scim:schemas:core:2.0",
         enterpriseSchema: "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User",
-        userSchema: "urn:ietf:params:scim:schemas:core:2.0:User"
+        userSchema: "urn:ietf:params:scim:schemas:core:2.0:User",
+        customEnterpriseSchema:"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User",
```
Comment (minolih) [inline apps/myaccount/src/extensions/configs/scim.ts:35]:
```suggestion
```

File: apps/myaccount/src/pages/account-security.tsx [hunk 2]
```diff
@@ -35,23 +39,43 @@ import {
...
 } from "../components";
 import { AppConstants, CommonConstants } from "../constants";
+import { commonConfig } from "../extensions";
+import { resolveUserStoreEmbeddedUsername } from "../helpers";
 import { InnerPageLayout } from "../layouts";
-import { AlertInterface, FeatureConfigInterface } from "../models";
+import { AlertInterface, AuthStateInterface, FeatureConfigInterface } from "../models";
 import { AppState } from "../store";
 import { addAlert } from "../store/actions";
+import {SCIMConfigs} from "../extensions/configs/scim";
```
Comment (minolih) [inline apps/myaccount/src/pages/account-security.tsx:48]:
```suggestion
import { SCIMConfigs } from "../extensions/configs/scim";
```

File: apps/myaccount/src/pages/overview.tsx [hunk 3]
```diff
@@ -16,51 +16,121 @@
...
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useSelector } from "react-redux";
 import { Overview } from "../components";
+import { commonConfig } from "../extensions";
 import { resolveUserProfileName } from "../helpers";
 import { InnerPageLayout } from "../layouts";
 import { AuthStateInterface } from "../models";
 import { AppState } from "../store";
+import {SCIMConfigs} from "../extensions/configs/scim";
```
Comment (minolih) [inline apps/myaccount/src/pages/overview.tsx:30]:
```suggestion
import { SCIMConfigs } from "../extensions/configs/scim";
```

**Review by minolih — APPROVED** [review]

**Review by minolih — APPROVED** [review]

**Review by NipuniBhagya — APPROVED** [review]

File: apps/myaccount/src/components/overview/widgets/profile-widget.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+ */
+export const ProfileWidget: FunctionComponent<AsgardeoProfileWidgetProps> = (
+    props
+): ReactElement => {
+
+    const {
+        userSource,
+        ["data-testid"]: testId
+    } = props;
+    const { t } = useTranslation();
```
Comment (brionmario) [inline apps/myaccount/src/components/overview/widgets/profile-widget.tsx:54]:
```suggestion

    const { t } = useTranslation();
```

File: apps/console/src/extensions/configs/models/common.ts [hunk 5]
```diff
@@ -1,23 +1,14 @@
 /**
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (brionmario) [inline apps/console/src/extensions/configs/models/common.ts:2]: Let's fix the license. Check other places as well.

File: apps/myaccount/src/components/overview/overview.tsx [hunk 6]
```diff
@@ -44,13 +45,19 @@ export const Overview: FunctionComponent<OverviewPropsInterface> = (
...
+        enableAlternateWidgetLayout
     } = props;
 
     const accessConfig: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features);
     const allowedScopes: string = useSelector((state: AppState) => state?.authenticationInformation?.scope);
 
-    const accountStatus = (widthComputer: SemanticWIDTHS, widthMobile: SemanticWIDTHS): React.ReactElement => {
+    /**
+     * Profile status widget with link to profile.
+     * @param widthComputer
```
Comment (brionmario) [inline apps/myaccount/src/components/overview/overview.tsx:57]: Check other places as well.

```suggestion
     * @param {SemanticWIDTHS} widthComputer - Width for the computer screens.
```

**Review by brionmario — APPROVED** [review]

**Pull request merged**